src/configs.rs

```diff
@@ -5,7 +5,15 @@ use clap::Clap;
 /// NEAR Indexer for Explorer
 /// Watches for stream of blocks from the chain
 #[derive(Clap, Debug)]
-#[clap(version = "0.1", author = "Near Inc. <hello@nearprotocol.com>")]
+#[clap(
+    version,
+    author = "Near Inc. <hello@nearprotocol.com>",
```

Comment:
This is enough:

```suggestion
    author,
```

clap will take the information from Cargo.toml :smile: (too smart, right?)

Reply:
Too smart, fixed 🥇